Src/BootCamp.Chapter/Calculate.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Calculate
+    {
+        private ILogger _logger;
+
+        public Calculate()
+        {
+            _logger = InitialiseLogger();
+            _logger.ActionLog("Calculation Program Started");
+        }
+        public double Add(params double[] numbers)
+        {
+            try
+            {
+                var sum = 0.0;
+                foreach (var x in numbers)
+                {
+                    if(x.GetType() == typeof(double))
```

Comment:
This is also wrong. You have an array of doubles. Every individual number is bound to be double

Reply:
Agreed